client/components/payment-method-logos/index.tsx

```diff
@@ -71,11 +71,27 @@ export const WooPaymentMethodsLogos: React.VFC< {
 	maxElements: number;
 } > = ( { maxElements = 10 } ) => {
 	const totalPaymentMethods = 21;
-	const maxElementsMiniView = 5;
+	const [ maxShownElements, setMaxShownElements ] = useState( maxElements );
+
+	useEffect( () => {
+		const getShowElements = () => {
```

Comment:
```suggestion
		const updateMaxElements = () => {
```
I believe we should avoid using the `get` prefix since it doesn’t return any value.

Reply:
changed in [6564ac2](https://github.com/Automattic/woocommerce-payments/pull/8977/commits/6564ac2b72c939d990a7ad759fb88aa2d8cef9d0) . thx